src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -362,16 +381,28 @@ public override void OnChildRemove(View view)
             mScrollingChild = new View();
         }
 
-        private void OnScrollStart()
+        private void OnScrollDragStart()
+        {
+            ScrollEventArgs eventArgs = new ScrollEventArgs();
```

Comment:
can I ask what kind of data are included in eventArgs ? it seems to new empty ScrollEventArgs object and set nothing.
then empty Args is sent to subscriber.

Reply:
For now, nothing needed. Just want to know when.